youtrack_sdk/client.py

```diff
@@ -133,6 +133,20 @@ def _post(
     def _delete(self, *, url: str) -> Optional[dict]:
         return self._send_request(method=HTTPMethod.DELETE, url=url)
 
+    def get_absolute_url(self, *, path: str) -> str:
+        return f"{self._base_url}{path}"
+
+    def issue_exists(self, *, issue_id: str) -> bool:
+        try:
+            self.get_issue(issue_id=issue_id)
+        except YouTrackNotFound:
+            return False
+        else:
+            return True
+
+    def get_issue_custom_field(self, *, issue: Issue, field_name: str) -> IssueCustomFieldType:
```

Comment:
I question the placement of this method
Maybe it should be in helpers instead as it has nothing to do with the `client`

Reply:
Done.